src/Symfony/Bundle/FrameworkBundle/Resources/config/lock.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Lock\LockFactory;
+use Symfony\Component\Lock\Store\CombinedStore;
+use Symfony\Component\Lock\Strategy\ConsensusStrategy;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->defaults()->private()
```

Comment:
It's already the case, please remove this defaults

Reply:
fixed it, thanks.